### DISCUSSION TITLE
Comment Snippets

### DISCUSSION THREAD

**Discussion by ThomDietrich** [openhab/openhab-docs#2049] [Team Posts]
Hey guys, @Confectrician asked about my "saved relies" on GitHub in the gitter channel. I only have one which is relevant for openhab-docs. Here it is:

**Comment by ThomDietrich** [comment]
```
Hello and thanks for your contribution! 👋 
Many parts of the documentation are [imported from other openHAB repositories](https://github.com/openhab/openhab-docs#automatically-generated-parts), e.g., the Binding- or add-on specific repository. This way we ensure that the articles are always in sync with the actual source code. This method has proven to be quite helpful to keep documentation up-to-date.
Long story short, you'll need to propose your change in the correct repository: 
<!--URL HERE-->

Thanks again for your willingness to contribute! 🙇
Please close this PR after you've created the new one.
```

**Comment by Confectrician** [comment]
Just for documentation, since i didn't know this yet:

https://blog.github.com/2016-03-29-saved-replies/
https://blog.github.com/2018-03-02-saved-replies-keyboard-shortcuts/

**Comment by bgilmer77** [comment]
I assume each of us would have to have our own saved replies.  So should we put candidates for saved replies here and then each of us can copy that into our own accounts?  Is that how it would work?  I don't have time right now to read up on this.

**Comment by Confectrician** [comment]
That's fully correct.
If you have answers you need to provide often, they can be stored there.
If i think there's somethign useful for al of us i will add it here.

**Comment by bgilmer77** [comment]
Well, clearly, the one that comes up over and over is making contributions in the -doc repo for adding, so yes, it would be good if we all give consistent replies for that.
